pkg/apis/release/v1alpha1/types.go

```diff
@@ -204,6 +207,33 @@ const (
 	ConditionPayloadRejected string = "PayloadRejected"
 )
 
+// ReleaseCreationJobResult houses the information about the Release creation batch job
+type ReleaseCreationJobResult struct {
+	// Coordinates the location of the job
```

Comment:
what kind of job is this?  `jobs.batch`?

Reply:
`batch/v1` Job

Reply:
update godoc and indicate which cluster it is present on?  The cluster running hte release controller?

Reply:
Done